### ISSUE TITLE
Sketchybar deps

### ISSUE THREAD

**Issue by marsjane** [john-json/dotfiles#3]
Thanks for the sharing, I saw you are using lua sketchybar config, and also helper folders, I assume you need all these following installed as deps, is it true?
```
[SbarLua](https://github.com/FelixKratz/SbarLua): A Lua API for SketchyBar
[sketchybar-app-font](https://github.com/kvndrsslr/sketchybar-app-font): A symbol font for SketchyBar
[SketchyBarHelper](https://github.com/FelixKratz/SketchyBarHelper): A header for C/C++ to directly communicate with SketchyBar
```

**Comment by john-json** [comment]
yes you need those for it to work with lua :) I would recommend installing sketchybar from the original lua installation and then just copy my folders over to your sketchybar folder.

**Comment by marsjane** [comment]
Oh does it have a lua version sketchybar？ Can you show me which one？ I only saw original sketchybar and the sbarlua for lua api

**Comment by john-json** [comment]
I don't know if im getting it all confused right now but here

[FelixKratz dotfiles](https://github.com/FelixKratz/dotfiles)


install it with `curl -L https://raw.githubusercontent.com/FelixKratz/dotfiles/master/install_sketchybar.sh | sh`

that's how I did it to install sketchybar with lua :)

**Comment by marsjane** [comment]
So nice of you! Much thanks i will try!
